doc/source/conf.py

```diff
@@ -334,9 +332,75 @@ def setup(app):
 
 #  --Options for sphinx extensions -----------------------------------------------
 
+# connect docs in other projects
 intersphinx_mapping = {
+    "pyproj": (
+        "https://pyproj4.github.io/pyproj/stable/",
+        "https://pyproj4.github.io/pyproj/stable/objects.inv",
+    ),
     "pandas": (
         "https://pandas.pydata.org/pandas-docs/stable/",
         "https://pandas.pydata.org/pandas-docs/stable/objects.inv",
     ),
-}
\ No newline at end of file
+    "shapely": (
+        "https://shapely.readthedocs.io/en/stable/",
+        "https://shapely.readthedocs.io/en/stable/objects.inv",
+    ),
+    "fiona": (
+        "https://fiona.readthedocs.io/en/latest/",
+        "https://fiona.readthedocs.io/en/latest/objects.inv",
+    ),
+    "pygeos": (
+        "https://pygeos.readthedocs.io/en/latest/",
+        "https://pygeos.readthedocs.io/en/latest/objects.inv",
+    ),
+    "rtree": (
+      "https://rtree.readthedocs.io/en/latest/",
+      "https://rtree.readthedocs.io/en/latest/objects.inv",
+    ),
+    "mapclassify": (
+        "https://pysal.org/mapclassify/",
+        "https://pysal.org/mapclassify/objects.inv"
+    ),
+    "libpysal": (
+        "https://pysal.org/libpysal/",
+        "https://pysal.org/libpysal/objects.inv"
+    ),
+    "matplotlib": (
+        "https://matplotlib.org/stable/",
+        "https://matplotlib.org/stable/objects.inv",
+    ),
+    "geopy": (
+        "https://geopy.readthedocs.io/en/stable/",
+        "https://geopy.readthedocs.io/en/stable/objects.inv",
+    ),
+    "cartopy": (
+        "https://scitools.org.uk/cartopy/docs/latest/",
+        "https://scitools.org.uk/cartopy/docs/latest/objects.inv"
+    ),
+    "pyepsg": (
+        "https://pyepsg.readthedocs.io/en/latest/",
+        "https://pyepsg.readthedocs.io/en/latest/objects.inv"
+    ),
+    "contextily": (
+        "https://contextily.readthedocs.io/en/latest/",
+        "https://contextily.readthedocs.io/en/latest/objects.inv"
+    ),
+    "rasterio": (
+        "https://rasterio.readthedocs.io/en/latest/",
+        "https://rasterio.readthedocs.io/en/latest/objects.inv"
+    ),
+    # Access denied error
+    # "geoplot": (
+    #     "https://docs.faculty.ai/libraries/geoplot/index.html",
```

Comment:
Try with https://residentmario.github.io/geoplot/index.html

Reply:
Thanks, I have [updated](https://github.com/geopandas/geopandas/pull/1931/commits/d2b4b9a006413c733924787a3a00f577422e712e) the config, it works now.